fms/utils/tensors.py

```diff
@@ -366,16 +534,45 @@ def _tensor(self):
     def __repr__(self):
         return self._tensor().__repr__()
 
-    # Since this is currently using a copy of the original
-    # tensor, these are all handled as out-of-place operations
-    # for now.
-    # Pointwise ops could all just be pass-through the the appropriate
-    # page of the underlying storage, but more complicated ops may need
-    # custom kernels.
+    # Most of these return a copy of the original
+    # tensor, but it handles pointwise ops with look-through
+    # into the underlying storage.
     @classmethod
     def __torch_function__(cls, func, types, args=(), kwargs=None):
         if kwargs is None:
             kwargs = {}
+
+        name = func.__qualname__.split(".")[-1]
```

Comment:
add comments to this chunk of code. 

Is this just getting the function to apply on a tensor as well as the arguments and applying them either inplace or out-of-place depending on the call?